net.sonobus.SonoBus.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: net.sonobus.SonoBus
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '20.08'
+command: sonobus
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=home
```

Comment:
Shouldn't this be just the "Music" folder by default, and maybe "Downloads"?

Reply:
good point, will do that instead

Reply:
they shall be specified with `xdg-music` and `xdg-downloads` instead of directories.

See https://docs.flatpak.org/en/latest/sandbox-permissions-reference.html?highlight=xdg-music#filesystem-permissions